src/lang/english.txt

```diff
@@ -1459,6 +1459,8 @@ STR_CONFIG_SETTING_VARIETY                                      :Variety distrib
 STR_CONFIG_SETTING_VARIETY_HELPTEXT                             :(TerraGenesis only) Control whether the map contains both mountainous and flat areas. Since this only makes the map flatter, other settings should be set to mountainous
 STR_CONFIG_SETTING_RIVER_AMOUNT                                 :River amount: {STRING2}
 STR_CONFIG_SETTING_RIVER_AMOUNT_HELPTEXT                        :Choose how many rivers to generate
+STR_CONFIG_SETTING_ROCKS_AMOUNT                                 :Rocks amount: {STRING}
```

Comment:
I think that the name should probably be clarified to emphasise that it's about the size of rocky patches, and not just more/less of the usual size rocky patches.

Reply:
```suggestion
STR_CONFIG_SETTING_ROCKS_AMOUNT                                 :Size of rocky patches: {STRING}
```

Something like this?

Reply:
Yes, I think that's better